org.eclipse.lemminx/pom.xml

```diff
@@ -267,5 +267,10 @@
 			<version>9.4.38.v20210224</version>
 			<scope>test</scope>
 		</dependency>
+		<dependency>
+			<groupId>org.commonmark</groupId>
+			<artifactId>commonmark</artifactId>
+			<version>0.17.1</version>
```

Comment:
I see a CQ of https://dev.eclipse.org/ipzilla/show_bug.cgi?id=22263 for 0.15.1 (before the groupId changed to org.commonmark), but we'll need to file a new one for this version.

Reply:
I will take a look into filing the CQ for it

Reply:
https://dev.eclipse.org/ipzilla/show_bug.cgi?id=23347

Reply:
From what I can tell, the CQ for commonmark is approved.